bin/cron/dms_tasks

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env ruby
+
+# This script runs the AWS Data Migration Service tasks that replicate data from
+# our reporting RDS database to our Redshift database. It is highly tied to the
+# AWS DMS tasks (https://console.aws.amazon.com/dms/home?region=us-east-1#tasks:)
+# and the CDO.dms_tasks secret.
+
+require 'cdo/hip_chat'
+
+# The list of DMS tasks to start, as well as the length of time that should
+# elapse before starting the next task (an overestimate of completion time).
+# The hash keys are expected to be keys in CDO.dms_tasks through which the task
+# ARNs are retrieved.
+TASKS = {
+  oneoff_survey_hierarchy: {
+    duration: 5
+  },
+  oneoff_user_progress_hierarchy: {
+    duration: 640
+  },
+  oneoff_teaching_hierarchy: {
+    duration: 10
+  },
+  oneoff_script_hierarchy: {
+    duration: 5
+  },
+  oneoff_user_hierarchy_pii: {
+    duration: 90
+  },
+  oneoff_plc_hierarchy: {
+    duration: 10
+  },
+  oneoff_user_proficiency_hierarchy: {
+    duration: 15
+  },
+  oneoff_contacts_hierarchy_pii: {
+    duration: 10
+  },
+  oneoff_multi_hierarchy: {
+    duration: 10
+  },
+  oneoff_metrics_hierarchy: {
+    duration: 10
+  },
+  oneoff_user_hierarchy: {
+    duration: 40
+  },
+  oneoff_hint_hierarchy: {
+    duration: 30
+  }
+}.freeze
+
+TASK_ARNS = CDO.dms_tasks.freeze
```

Comment:
If the names of all tasks are already in this document, you should be able to derive the respective ARNs through the [DescribeReplicationTasks](http://docs.aws.amazon.com/dms/latest/APIReference/API_DescribeReplicationTasks.html) API (using e.g., either [`describe-replication-tasks`](http://docs.aws.amazon.com/cli/latest/reference/dms/describe-replication-tasks.html) CLI or [`describe_replication_tasks`](http://docs.aws.amazon.com/sdkforruby/api/Aws/DatabaseMigrationService/Client.html#describe_replication_tasks-instance_method) AWS Ruby SDK), there shouldn't be any need to duplicate them through CDO config variables.